main.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+function calculateAge(date) {

```

Comment:
Good! Also, you can use a static method for this propose
Not need to rewrite, just FYI